sdk/provisioning/Azure.Provisioning.CloudMachine/src/CDKLevel3/CloudMachineInfrastructure.cs

```diff
@@ -309,7 +309,7 @@ public static bool Configure(string[] args, Action<CloudMachineInfrastructure>?
                 if (name.StartsWith("I")) name = name.Substring(1);
                 string directory = Path.Combine(".", "tsp");
                 string tspFile = Path.Combine(directory, $"{name}.tsp");
-                Directory.CreateDirectory(Path.GetDirectoryName(tspFile));
+                Directory.CreateDirectory(Path.GetDirectoryName(tspFile)!);
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/provisioning/Azure.Provisioning/src/Azure.Provisioning.csproj#L7